readme.md

```diff
@@ -7,13 +7,14 @@ languages:
 - bicep
 products:
 - azure
+- azure-openai
 - azure-container-apps
 - azure-container-registry
 - azure-cosmos-db
 - blazor
 - aspnet-core
 - entra-id
-urlFragment: template
+urlFragment: chat-csharp-cosmos-db-nosql-openai
```

Comment:
```suggestion
urlFragment: template
```

Is there a reason that we're repeating the name of the repository in the URL fragment?

With this configuration, the final URL would be: <https://learn.microsoft.com/samples/azure-samples/chat-csharp-cosmos-db-nosql-openai/chat-csharp-cosmos-db-nosql-openai>. I would prefer for it to be: <https://learn.microsoft.com/samples/azure-samples/chat-csharp-cosmos-db-nosql-openai/template> since it's an AD template